docs/recipes.rst

```diff
@@ -55,6 +55,21 @@ You can also provide multiple headers by using the ``-H`` option multiple times:
 
     schemathesis run -H "Authorization: ..." -H "X-API-Key: ..."
 
+
+It is possible to authenticate with an unencrypted certificate (e.g. PEM) by using the ``--request-cert /file/path/example.pem`` argument.
+
+.. code-block:: text
+
+    schemathesis run --base-url https://example.com --request-tls-verify=no --request-cert client.pem /app/open-api.yaml
```

Comment:
I think that it would be slightly more readable if we remove `--base-url https://example.com`  and replace `/app/open-api.yaml` with `...` as in other CLI examples above. A similar comment goes to the example below.

Reply:
Fixed.